[2024-03-26 06:07:22,242] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: bigdag.convert_xml_to_data_task 1a32181c-53b6-4435-8794-b63e86d3a490 [queued]>
[2024-03-26 06:07:22,322] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: bigdag.convert_xml_to_data_task 1a32181c-53b6-4435-8794-b63e86d3a490 [queued]>
[2024-03-26 06:07:22,325] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 06:07:22,326] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-03-26 06:07:22,331] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 06:07:22,410] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): convert_xml_to_data_task> on 2024-03-26 06:06:02.981622+00:00
[2024-03-26 06:07:22,433] {standard_task_runner.py:52} INFO - Started process 339 to run task
[2024-03-26 06:07:22,463] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'bigdag', 'convert_xml_to_data_task', '1a32181c-53b6-4435-8794-b63e86d3a490', '--job-id', '1325', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpz5syenro', '--error-file', '/tmp/tmpejdlc7ga']
[2024-03-26 06:07:22,480] {standard_task_runner.py:80} INFO - Job 1325: Subtask convert_xml_to_data_task
[2024-03-26 06:07:22,825] {task_command.py:369} INFO - Running <TaskInstance: bigdag.convert_xml_to_data_task 1a32181c-53b6-4435-8794-b63e86d3a490 [running]> on host 911e7a0235cb
[2024-03-26 06:07:23,200] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bigdag
AIRFLOW_CTX_TASK_ID=convert_xml_to_data_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-26T06:06:02.981622+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=1a32181c-53b6-4435-8794-b63e86d3a490
[2024-03-26 06:07:23,241] {logging_mixin.py:115} INFO - /opt/***/dags/resources/1a32181c-53b6-4435-8794-b63e86d3a490/Level1_combined.csv
[2024-03-26 06:07:23,243] {logging_mixin.py:115} INFO - /opt/***/dags/resources/1a32181c-53b6-4435-8794-b63e86d3a490/Level1_combined.grobid.tei.xml
[2024-03-26 06:07:23,245] {logging_mixin.py:115} INFO - ['/opt/***/dags/resources/1a32181c-53b6-4435-8794-b63e86d3a490/Level1_combined.grobid.tei.xml']
[2024-03-26 06:07:23,246] {logging_mixin.py:115} INFO - /opt/***/dags/resources/1a32181c-53b6-4435-8794-b63e86d3a490/Level1_combined.grobid.tei.xml
[2024-03-26 06:07:23,249] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/tasks/convert_to_csv.py", line 14, in convert_xml_to_data_task
    return convert_xml_to_csv(run_id,data)
  File "/opt/airflow/dags/helpers/xml_to_csv_helper.py", line 27, in convert_xml_to_csv
    with open(xml_file_path, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/dags/resources/1a32181c-53b6-4435-8794-b63e86d3a490/Level1_combined.grobid.tei.xml'
[2024-03-26 06:07:23,279] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=bigdag, task_id=convert_xml_to_data_task, execution_date=20240326T060602, start_date=20240326T060722, end_date=20240326T060723
[2024-03-26 06:07:23,308] {standard_task_runner.py:97} ERROR - Failed to execute job 1325 for task convert_xml_to_data_task ([Errno 2] No such file or directory: '/opt/airflow/dags/resources/1a32181c-53b6-4435-8794-b63e86d3a490/Level1_combined.grobid.tei.xml'; 339)
[2024-03-26 06:07:23,333] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-03-26 06:07:23,516] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
